client/asset/btc/btc.go

```diff
@@ -449,6 +449,15 @@ func (btc *ExchangeWallet) ReturnCoins(unspents asset.Coins) error {
 	return btc.wallet.LockUnspent(true, ops)
 }
 
+// DecodeCoinID creates a human-readable representation of a coin ID.
+// func (dcr *ExchangeWallet) DecodeCoinID(coinID dex.Bytes) (string, error) {
+// 	txHash, vout, err := decodeCoinID(coinID)
+// 	if err != nil {
+// 		return "", err
+// 	}
+// 	return fmt.Sprintf("%v:%d", txHash, vout), err
+// }
```

Comment:
So after reviewing what I was doing here, I think I kept this in draft because I wasn't sure how to handle decoding here https://github.com/decred/dcrdex/pull/271#discussion_r409682859 without such an ExchangeWallet method.  What do you think @buck54321?  I think we need an easy way to decode a coin ID if you don't have an `asset.Coin`. Maybe there's a better way to handle this.

Reply:
I looked at it earlier and didn't see a clear solution. Seems like we keep running up against it. If that's the case, we can just mimic the server **Driver.DecodeCoinID** method on the client's driver and do it that way.

Reply:
I guess the method is fine on the `ExchangeWallet` too. I only suggested the `Driver` because then decoding wouldn't even require a wallet to be created first.

Reply:
Went with the driver function.  It's not an issue here, but it would require importing asset.